HealthMonitoring.SelfHost/Content/Scripts/homeController.js

```diff
@@ -23,29 +26,68 @@
         var configFrequency = getConfigUpdatingFrequency($location);
 
         $scope.displayHint = function (evnt) {
-            if ($scope.statLegend == null) {
+            if ($scope.statLegend === null) {
                 $scope.statLegend = {
                     status: evnt.target.innerHTML,
                     description: statusLegend[evnt.target.innerHTML],
                     x: evnt.clientX + 5,
                     y: evnt.clientY + 5
                 };
             }
-        }
+        };
+
+        $scope.getGroupColour = function (group) {
```

Comment:
The color response, is an aggregate - in order of priority. e.g. TimedOut takes preference over UnHealthy. We use the Status(es) column to identify the Group Status distribution.